java/ingest/src/main/java/sleeper/ingest/job/IngestJobQueueConsumer.java

```diff
@@ -248,13 +179,13 @@ public long ingest(IngestJob job, Schema schema, TableProperties tableProperties
                 .withNamespace(metricsNamespace)
                 .withMetricData(new MetricDatum()
                         .withMetricName("StandardIngestRecordsWritten")
-                        .withValue((double) numRecordsWritten)
+                        .withValue((double) result.getNumRecordsWritten())
                         .withUnit(StandardUnit.Count)
                         .withDimensions(
                                 new Dimension().withName("instanceId").withValue(instanceId),
                                 new Dimension().withName("tableName").withValue(job.getTableName())
                         )));
 
-        return numRecordsWritten;
+        return result.getNumRecordsWritten();
```

Comment:
This method computes the value rather than just returning it. We're calling it three times right now. We could assign the returned value to a local variable or precompute it.

Reply:
Oops, I'm wrong. I think I was thinking of another version of the code. Let me have another look.